### ISSUE TITLE
Set working directory

### ISSUE THREAD

**Issue by MartijnBNG** [phips28/gh-action-bump-version#4]
Hi! I've been using this action to bump my versions and it's very useful! In one of our repos however, our package.json is not in the root dir.

So I'd like to request a feature where I can set the working-directory for this action. I can't write js, so I can't create a PR, but what I have in mind is along these lines:

In my push.yml I'd add the last line of this:
```
      - name: "Automated Version Bump"
        uses: "phips28/gh-action-bump-version@master"
        with:
          tag-prefix: ''
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          PACKAGEJSON_DIR: /src/static
```

And in index.js at line 6, you'd insert the js equivalent of this pseudocode:
```
if (isNotEmpty(process.env.working-directory))
    setWorkingDirectory(process.env.working-directory)
```

Would something like this be possible?

**Comment by MartijnBNG** [comment]
I decided to try anyway, no idea if this is good js code:
https://github.com/phips28/gh-action-bump-version/pull/5

**Comment by MartijnBNG** [comment]
I've tried this with our repo (unfortunately private) and this works for us. Behavior when PACKAGEJSON_DIR is undefined remains the same.

**Comment by phips28** [comment]
closed #6
